lib/convection/model/template/resource.rb

```diff
@@ -337,10 +337,11 @@ def reference
           }
         end
 
-        def with_output(output_name = name, value = reference, &block)
+        def with_output(output_name = name, value = reference, export_as: nil, &block)
```

Comment:
Is there any reason to have this named parameter? The user can just add the output in the existing configuration block parameter.